R/class-workbook.R

```diff
@@ -4188,8 +4281,7 @@ wbWorkbook <- R6::R6Class(
     },
 
     ws = function(sheet) {
-      sheet_id <- wb_validate_sheet(self, sheet)
-      self$worksheets[[sheet_id]]
+      self$worksheets[[private$get_sheet(sheet)]]
```

Comment:
Ah! Seeing this, I once wanted to use this as a replacement or shortcut in my code.
So that 
```R
wb$worksheets[[sheet]] ...
```
would become
```R
wb$ws(sheet) ...
```
but my poor man's attempt obviously failed. If this cannot be achieved we can remove this function here and in `wb_functions.R`. most likely it misses a `ws<-` function